packages/plugin-ext/src/plugin/tasks/tasks.ts

```diff
@@ -109,6 +109,27 @@ export class TasksExtImpl implements TasksExt {
         return this.createDisposable(callId);
     }
 
+    async fetchTasks(filter?: theia.TaskFilter): Promise<theia.Task[]> {
+        const taskVersion = filter ? filter.version : undefined;
+        const taskType = filter ? filter.type : undefined;
+        const taskDtos = await this.proxy.$fetchTasks(taskVersion, taskType);
+        return Promise.resolve(taskDtos.map(dto => converter.toTask(dto)));
```

Comment:
just fyi: you don't need `Promise.resolve` or `Promise.reject` in async context

just `return` or `throw`

Reply:
thank you for the review. fixed.